clients/spark/core/src/main/scala/io/treeverse/clients/LakeFSContext.scala

```diff
@@ -66,6 +66,9 @@ object LakeFSContext {
   val LAKEFS_CONF_DEBUG_GC_MAX_COMMIT_ISO_DATETIME_KEY = "lakefs.debug.gc.max_commit_iso_datetime"
   val LAKEFS_CONF_DEBUG_GC_MAX_COMMIT_EPOCH_SECONDS_KEY = "lakefs.debug.gc.max_commit_epoch_seconds"
   val LAKEFS_CONF_DEBUG_GC_REPRODUCE_RUN_ID_KEY = "lakefs.debug.gc.reproduce_run_id"
+
+  //  Objects that are written during the skip period are not collected
+  val LAKEFS_CONF_DEBUG_GC_UNCOMMITTED_SKIP_SECONDS_KEY = "lakefs.debug.gc.uncommitted_skip_seconds"
```

Comment:
is this key only for debug? Otherwise drop the "debug" word.
Also, how about "gc.uncommitted_min_age_seconds" as the name?

Reply:
I believe this should be used only for debugging. The default value should be tied to some server-internal configuration, like the validity duration of the token used for Get/LinkPhysicalAddress. Making this anything other than debug/plumbing is allowing an unaware user to potentially delete unlinked data.

Regarding the name - sounds better, changing.